support-frontend/knip.config.js

```diff
@@ -1,11 +1,11 @@
 const entryPoints = require('../webpack.entryPoints').common;
 
 const flattenedEntryPoints = [...Object.keys(entryPoints)].map(
-	(key) => `assets/${entryPoints[key]}`,
+	(key) => `assets/${entryPoints[key]}!`,
```

Comment:
So the `!` here is a signal to knip that this is production code?

Reply:
Exactly, is it worth a comment or [link](https://knip.dev/features/production-mode#:~:text=and%20project%20patterns-,suffixed%20with%20!,-Only%20production%20entry)?

Reply:
Yeah I think a comment/link would be good 👍